buildenv/docker/mkdocker.sh

```diff
@@ -916,6 +919,10 @@ fi
 }
 
 main() {
+if [ $dist$version = centos6 -a $arch = x86_64 ] ; then
+   # no cert check was needed and verified required for centos6 x86_64
+   wget_O="wget --no-check-certificate --progress=dot:mega -O"
+fi
```

Comment:
Maybe a better place for this is after line 171 that applies only to centos6:
```
          if [ $arch = x86_64 ] ; then
            # Certificates are old on CentOS:6 so we can't expect wget to check.
            wget_O="wget --progress=dot:mega --no-check-certificate -O"
          fi
```

Reply:
Update made.